### DISCUSSION TITLE
Extruding text

### DISCUSSION THREAD

**Discussion by LegNeato** [bevyengine/bevy#5235] [Q&A] [answered]
Is there any way to take text and extrude it into a geometry, a la https://youtu.be/oc2_iWJfUUM? I'd like to have text be 3d and interact with my 3 scene without manually modeling anything.

**Comment by NiklasEi** [comment, marked as answer]
There is a community plugin to generate meshes from text: https://github.com/blaind/bevy_text_mesh

**Reply by LegNeato** [reply]
Thank you!
